ffmpeg@4.4.1-2: download via aria2 failed

>cls && scoop uninstall ffmpeg; scoop update; scoop install ffmpeg;

Uninstalling 'ffmpeg' ().
'ffmpeg' was uninstalled.
Updating Scoop...
Updating 'extras' bucket...
Updating 'games' bucket...
Updating 'java' bucket...
Updating 'jetbrains' bucket...
Updating 'main' bucket...
Updating 'nerd-fonts' bucket...
Updating 'nightlies' bucket...
Updating 'nirsoft' bucket...
Updating 'nonportable' bucket...
Updating 'php' bucket...
Updating 'versions' bucket...
Scoop was updated successfully!
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'ffmpeg' (4.4.1-2) [64bit]
Starting download with aria2 ...
Download: 11/21 02:58:00 [ERROR] CUID#8 - Download aborted. URI=https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-06-12-24/ffmpeg-n4.4.1-2-gcc33Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-06-12-24/ffmpeg-n4.4.1-2-gcc33e7Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 6ca2fd|ERR |       0B/s|C:/Users/Akira/scoop/cache/ffmpeg#4.4.1-2#https_github.com_BtbN_FFmpeg-Builds_releases_download_autobuild-2021-11-06-12-24_ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-06-12-24/ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4.zip
    referer=https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-06-12-24/
    dir=C:\Users\Akira\scoop\cache
    out=ffmpeg#4.4.1-2#https_github.com_BtbN_FFmpeg-Builds_releases_download_autobuild-2021-11-06-12-24_ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4.zip

ERROR & 'C:\Users\Akira\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\Akira\scoop\cache\ffmpeg.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=9532 --continue --summary-interval 0

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=ffmpeg%404.4.1-2%3a+download+via+aria2+failed

--- a/bucket/ffmpeg-shared.json
+++ b/bucket/ffmpeg-shared.json
@@ -5,8 +5,8 @@
     "license": "GPL-3.0-or-later",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-06-12-24/ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-shared-4.4.zip",
-            "hash": "332516c12364e4758ac092c715f6bea19b8220bae2253964516d5f2f6cd4d715",
+            "url": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-22-12-21/ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-shared-4.4.zip",
+            "hash": "c41e3a4d39a6752f57e74f39bc32e18fa6646edebe6a9223f8a440f537cba670",
             "extract_dir": "ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-shared-4.4"
         }
     },

--- a/bucket/ffmpeg.json
+++ b/bucket/ffmpeg.json
@@ -4,7 +4,7 @@
     "homepage": "https://ffmpeg.org",
     "license": "GPL-3.0-or-later",
     "architecture": {
-        "64bit": { 
+        "64bit": {
             "url": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-22-12-21/ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4.zip",
             "hash": "4b96a3fd21e8f6d1b088c32ff6d88bbeb05ae343dbad2d58b405e33cda2090c5",
             "extract_dir": "ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4"

--- a/bucket/ffmpeg.json
+++ b/bucket/ffmpeg.json
@@ -4,9 +4,9 @@
     "homepage": "https://ffmpeg.org",
     "license": "GPL-3.0-or-later",
     "architecture": {
-        "64bit": {
-            "url": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-06-12-24/ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4.zip",
-            "hash": "9e146e7bcd39995f8a09e00ebb5866b3f86ac7e183c4218a7fa263fbcdc94ddc",
+        "64bit": { 
+            "url": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-11-22-12-21/ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4.zip",
+            "hash": "4b96a3fd21e8f6d1b088c32ff6d88bbeb05ae343dbad2d58b405e33cda2090c5",
             "extract_dir": "ffmpeg-n4.4.1-2-gcc33e73618-win64-gpl-4.4"
         }
     },

